frontend/README.md

```diff
@@ -5,51 +5,61 @@ This directory (`frontend`) contains the frontend service for the Pathways-based
 NOTE: All commands in this README are written with the assumption that the working directory is `frontend`, e.g. you have used `cd frontend` from the root directory of the pbtar repo, so that the context is exclusive to the Frontend service.
 
 ## Set-Up
+
 This project uses Node.js and npm for dependency management.
 
 To install Node.js, follow the [official installation guide](https://nodejs.org/en/download/).
 
 1. Clone the Repo
-``` bash
+
+```bash
 git clone https://github.com/your-username/pbtar.git
 cd pbtar/frontend
 ```
 
 2. Install Dependencies
-``` bash
+
+```bash
 npm install
 ```
 
 3. Running the Frontend
-You can locally service the frontend *alone* with:
-``` bash
+   You can locally service the frontend _alone_ with:
+
+```bash
 npm run dev # which calls vite
 ```
 
 The local development service will be accessible at `http://localhost:3000`. It automatically reloads on file changes.
```

Comment:
```suggestion
The local development service will be accessible at `http://localhost`. It automatically reloads on file changes.
```

Reply:
When running the "local" development service with `npm run dev`, it is available at `3000`. 
It's only through `docker compose` that it appears at `80`.

Reply:
Trying to run the `vite` server itself on `80` seems to cause problems:
``` bash
error when starting dev server:
Error: listen EACCES: permission denied ::1:80
    at Server.setupListenHandle [as _listen2] (node:net:1800:21)
    at listenInCluster (node:net:1865:12)
    at GetAddrInfoReqWrap.doListen [as callback] (node:net:2014:7)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:110:8)
```

Reply:
hmmmm... ok